docker-compose.yml

```
@@ -0,0 +1,44 @@
+version: "3"
+
+services:
+  dev:
+    platform: linux/amd64
+    build:
+      context: .
+      dockerfile: ./Dockerfiles/Dockerfile.todoapp
+      target: dev
+    volumes:
+      - type: bind
+        source: ./todo_app
+        target: /app/todo_app
```

Less easy to read arguably, but worth noting that the concise syntax is an option:
```suggestion
      - ./todo_app:/app/todo_app
```

The fact it starts with a path rather than name means it knows it's a bind mount